src/components/SearchBar.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const SearchBar = ({ searchTerm, setSearchTerm }) => {
+  const handleChange = (event) => {
+    setSearchTerm(event.target.value);
+  };
+
+  const clearText = (event) => {
+    event.preventDefault();
+    setSearchTerm('');
+  };
+
+  return (
+    <div>
+      <input
+        type="text"
+        placeholder="Search list.."
+        value={searchTerm}
+        onChange={handleChange}
+      />
+      <button type="reset" onClick={clearText} value="Reset">
+        X
+      </button>
```

Comment:
Just pointing the last comment here in the exact spot. 

_This is the same comment I left in the previous discussion:_

There is one more detail I should mention.

As the AC asks, we need to conditionally render the "X" button.
Just a hint: Maybe check if `searchTerm` has a value. If it does not, then don't render the button. Otherwise, render it.
Please let me know if you have any questions 👍

Reply:
Ah great, I see! I wanted to do this in the styling session in the last week anyway :)